src/BlazorUI/Bit.BlazorUI/Components/Progress/Shimmer/BitShimmer.scss

```
@@ -35,43 +49,114 @@
+    animation-delay: var(--bit-smr-dly, 0ms);
+    animation-duration: $mot-duration-long;
+    animation-timing-function: $mot-easing;
+    animation-name: bit-smr-anim-opacity;
+}
+
 .bit-smr-wrp {
     height: 100%;
     overflow: hidden;
     position: relative;
     animation-fill-mode: both;
+    // Held at zero opacity for as long as ShowDelay asks for, so a call that finishes inside that window is
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)